albatross/core/model_adapter.h

```diff
@@ -117,6 +117,38 @@ class AdaptedRegressionModel
     return sub_model_.get_fit();
   }
 
+  /*
+   * Cross validation specializations
+   */
+  virtual std::vector<JointDistribution> cross_validated_predictions_(
```

Comment:
Maybe leave an informational comment lying around somewhere explaining about why these methods have the trailing underscore -- name resolution / ADL rules yada yada yada. Comments like this don't really run the risk of turning "stale" if they are purely informational and not necessarly specific to the code at hand.

Reply:
yeah good call.